docs/source/quicktour.mdx

```diff
@@ -264,10 +268,10 @@ tensor([[0.0021, 0.0018, 0.0115, 0.2121, 0.7725],
 >>> import tensorflow as tf
 
 >>> tf_predictions = tf.nn.softmax(tf_outputs.logits, axis=-1)
->>> print(tf_predictions)
+>>> print(tf.math.round(tf_predictions * 10**4) / 10**4)
```

Comment:
No, doctest will round all numbes to the precision written, whether inside tensors or not. This passed locally before so should still pass by tweaking the display. We should avoid cluttering the example for the sake of the doctests and can just ignore the output of this particular test if needs be.

Reply:
Didn't pass for me locally - could you maybe quickly double check here? Note that all tests were passing locally before because we were ignoring the output.

If they pass locally for you, happy to revert it. PT passes because they automatically round to 4 decimal places

Reply:
Happy to fix in a follow-up PR! Didn't manage to make it work without rounding (neither locally nor on the ci-runner, see message offline).